ChangeLog

```diff
@@ -49,6 +49,10 @@ Bugfix
    * Fix propagation of restart contexts in restartable EC operations.
      This could previously lead to segmentation faults in builds using an
      address-sanitizer and enabling but not using MBEDTLS_ECP_RESTARTABLE.
+   * Fix incorrect recovery of internal record header pointers during context
+     deserialization. This previously lead to failure when attempting to write
+     a record after context deserialization if either the Connection ID
+     extension or GCM or CCM were in use.
```

Comment:
Should we add bug fix notes to the ChangeLog for features not shipped in any released version?

Reply:
You tell me.